frontend/public/components/factory/ListPage/ListPageCreate.tsx

```diff
@@ -104,9 +104,14 @@ const ListPageCreate: React.FC<ListPageCreateProps> = ({
     }
   }
 
+  const accessReview: CreateWithPermissionsProps['createAccessReview'] = createAccessReview || {
```

Comment:
This should actually be `ListPageCreateLinkProps` since that's the component we use below.

```suggestion
  const accessReview: ListPageCreateLinkProps['createAccessReview'] = createAccessReview || {
```